changelog.d/1402.misc.md

```diff
@@ -0,0 +1 @@
+Enable and apply ruff rule `ISC001`.
```

Comment:
Since this does not affect end users, there is no need to have a changelog.

Reply:
I have removed `changelog.d/1402.misc.md`.

You might want to remove `changelog.d/1400.misc.md` as well.